packages/core/src/shared/sam/build.ts

```diff
@@ -281,3 +283,13 @@ function resolveBuildArgConflict(boundArgs: string[]): string[] {
     // }
     return Array.from(boundArgsSet)
 }
+export async function resolveBuildFlags(buildFlags: string[], samCliVersion: SemVer | null): Promise<string[]> {
+    // const samCliVersion = (await getSamCliPathAndVersion()).parsedVersion
+    if (samCliVersion?.compare('1.133.0') ?? -1 < 0) {
```

Comment:
what are these magic numbers telling us? Might want a comment for anyone who looks at this in the future